deploy/examples/crds.yaml

```diff
@@ -12377,6 +12377,13 @@ spec:
                 phase:
                   description: ConditionType represent a resource's status
                   type: string
+                replicas:
+                  format: int32
+                  nullable: true
+                  type: integer
+                selector:
+                  nullable: true
+                  type: string
```

Comment:
This is not part of my review, just an overall feauture note. From reading the scale subresource doc, I can see that this is supposed to be a simple string type, but I have to admit that I confused about how this is expected to work IRL. The doc doesn't seem to explain that well from what I see. Do you understand why this is a simple string instead of a list of selectors?